scripts/airgap.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# ==================================================================================================
+# TODO:
+#
+# Review (Victor): https://github.com/kubewarden/kwctl/pull/375#discussion_r1048740479
+# current approach doesn't use the GH release files kubewarden-crds-1.2.3_kubewarden-defaults-1.2.8_kubewarden-controller-1.2.8_policylist.txt
+# - it does
+# canonical locations of artifacts
+# - please explain (helm charts vs release?)
+#
+# TODO:
+# fix https://docs.kubewarden.io/operator-manual/airgap/requirements
+# blogpost - https://github.com/kubewarden/kubewarden.io/pull/153
+# fix documentation - https://github.com/kubewarden/docs/pull/150
+# - get image / policy list from pulled charts?
+# ==================================================================================================
+
+
+# Defaults
+IMAGES_TAR="kubewarden-images.tar"
+POLICIES_TAR="kubewarden-policies.tar.gz"
+
+PROG=$(basename "$0")
+USAGE="\
+Usage:  $PROG command [--parameters]
+
+  $PROG list [--cert-manager]
+  $PROG pull -l file.json [--dry]
```

Comment:
I wonder about using json as the image list format, do we know if it would be useful for any specific tool? 
Maybe is easier to just have a line list as it was. It also makes it as the output used in similar tools by the community, such as the [helm image plugin](https://github.com/cvila84/helm-image).